plugins/exporter.koplugin/clip.lua

```diff
@@ -230,26 +230,50 @@ function MyClipping:getImage(image)
     local doc = DocumentRegistry:openDocument(image.file)
     if doc then
         local png = doc:clipPagePNGString(image.pos0, image.pos1,
-                image.pboxes, image.drawer)
+            image.pboxes, image.drawer)
         --doc:clipPagePNGFile(image.pos0, image.pos1,
-                --image.pboxes, image.drawer, "/tmp/"..md5(png)..".png")
+        --image.pboxes, image.drawer, "/tmp/"..md5(png)..".png")
         doc:close()
         if png then return { png = png, hash = md5(png) } end
     end
 end
 
+local function sortByPosition(a, b)
```

Comment:
I'm afraid it's not so easy. Look at
https://github.com/koreader/koreader/blob/0808560532eff82eb3495bd8b378d76802dec1fe/frontend/apps/reader/modules/readerbookmark.lua#L198

Reply:
True, it is not.